docs/decisions/0005-user-and-group-management-commands.rst

```diff
@@ -0,0 +1,19 @@
+User and Group Management Commands
+===============================
+
+Context
+-------
+
+Open edX administrators ocassionally need to grant permissions to users for certain django services (e.g. ecommerce, registrar). Right now, this is done manually via django admin of the django app. This method of granting permissions is not endorsed.
+
+The user and group management commands in edx-platform, are supposedly generic enough to be used via a common library/app (such as edx-django-utils) so that users and groups can be managed without much modification to IDAs.
```

Comment:
```suggestion
edx-platform, however, defines ``manage_user`` and ``manage_group`` management commands, which allow users to be managed via an external system (such as one, for example, that defines permissions declaratively in version-controlled YAML). These user and group management commands are supposedly generic enough to be used via a common library/app (such as edx-django-utils) so that this users and group management scheme can be brought to other IDAs.
```

Suggested additional context